Samples/Server/NativeServer.Tests/NativeServersTests.cpp

```diff
@@ -442,3 +445,81 @@ TEST(BufferCapturerTests, CaptureFrameStereoUsingDirectXBufferCapturer)
 			capturer->staging_frame_buffer_.Get(), 0);
 	}
 }
+
+// --------------------------------------------------------------
+// Decoder tests
+// --------------------------------------------------------------
+TEST(DecoderTests, CanDecodeCorrectly) {
```

Comment:
Can you rename the unit test to RtpHeaderFramePredictionTest? That is the core of this test not the decode itself, we have plenty of decoder tests inside webrtc.

Reply:
Done!